### PR TITLE
Add progress bar

### PR DESCRIPTION
**Pull request by mmerrittsmith**

### CODE REVIEW

**Review by satejsoman — CHANGES_REQUESTED** [review]
can we add this behind a flag? slurm didn't get along with the multiprocessing dependencies behind tqdm last time i played around with this - and more generally i don't think we should assume a TTY is available (but not going to block this PR on this)

**Comment by mmerrittsmith** [comment]
Yeah of course!

**Comment by mmerrittsmith** [comment]
Ok that oughta do it

File: prclz/etl/download.py [hunk 1]
```diff
@@ -27,10 +27,16 @@ def build_data_dir(root: str, additional: Optional[Sequence[str]] = None) -> Dic
 
     return data_paths
 
-def download(src: URL, dst: Path) -> None:
-    r = requests.get(src, stream = True)
+def download(src: URL, dst: Path, verbose) -> None:
```
Comment (satejsoman) [inline prclz/etl/download.py]: type annotation?

**Review by satejsoman — APPROVED** [review]

**Pull request merged**